manifests/server.pp

```diff
@@ -92,15 +92,21 @@
     },
 
     exec { 'pds-migrations':
-      unless  => '/opt/puppetlabs/sbin/pds-ctl rake db:migrate:status',
       command => Sensitive(@("CMD"/L)),
         /usr/bin/test "$(/opt/puppetlabs/sbin/pds-ctl rake db:version | cut -d ':' -f 2)" -eq 0 && \
-        /opt/puppetlabs/sbin/pds-ctl rake db:migrate && \
-        /opt/puppetlabs/sbin/pds-ctl rake 'app:set_admin_token[${pds_token.unwrap}]'
+        /opt/puppetlabs/sbin/pds-ctl rake db:migrate
         | CMD
       require => Class['puppet_data_service::anchor'],
     },
 
+    exec { 'pds-set-admin-user':
+      unless  => '/opt/puppetlabs/sbin/pds-ctl rake app:does_admin_exist',
```

Comment:
@reidmv I'm using the[ new rake task introduced in this PR](https://github.com/puppetlabs/puppet-data-service/pull/68)